templates/server/index.html

```diff
@@ -1,460 +1,484 @@
+    background-position: 0% 0%, top right, right 0 bottom 4rem, right bottom, 0% 0%;
+    background-repeat: no-repeat;
+    background-size: 100% calc(100% - 4rem),50% 100%,100% 4rem,100% 4rem,auto;
+    margin-bottom: -4rem;
+    overflow: hidden;
+    padding-bottom: 12rem !important;
+    position: relative;
+}
+.p-strip--suru.is-dark {
+    color: #fff;
+}
+.fiximg {
+  padding-top: 4rem
+}
+.chart {
+	margin-top: 1.5rem
+}
+.chart__bar--orange {
+    fill: #e95420;
+    stroke: #e95420;
+    stroke-width: 1px;
+}
+.chart__bar--orange-light {
+    fill: #fbddd2;
+    stroke: #e95420;
+    stroke-width: 1px;
+}
+.chart__bar--aubergine {
+    fill: #772953;
+    stroke: #772953;
+    stroke-width: 1px;
+}.chart__bar--grey {
+    fill: #aea79f;
+    stroke: #aea79f;
+    stroke-width: 1px;
+}
+.chart__label--bold {
+	font-weight: bold
+}
+.chart-key {
+	display: block
+}
+.chart-key__row {
+	display: block
+}
+.chart-key text {
+	font-size: 14px
+}
+.p-bar-chart {
+	width: 100%
+}
+.p-bar-chart .p-bar-chart__bar {
+	opacity: 1
+}
+.p-bar-chart .label {
+	font-weight: 400
+}
+.p-bar-chart .tick text {
+	font-size: .625rem
+}
+.p-bar-chart .left-axis {
+	font-size: 1rem;
+	text-align: right
+}
+.p-progress-chart {
+	background: #fff;
+	border: 1px solid #cdcdcd;
+	box-shadow: 0 1px 5px 1px rgba(17,17,17,0.2)
+}
+.p-progress-chart .p-progress-chart__bar {
+	opacity: 1
+}
+.p-progress-chart .p-progress-chart__text {
+	font-size: 2.91029rem
+}
+.p-pie-chart .p-pie-chart__wedge path {
+	opacity: 1
+}
+.p-pie-chart .p-pie-chart__text {
+	font-size: 1.2rem
+}
+@media only screen and (max-width: 875px) {
+	.p-pie-chart .p-pie-chart__text {
+		font-size: 1rem
+	}
+}
+.x.axis line {
+	shape-rendering: crispEdges;
+	stroke: #d6d3cf;
+	stroke-dasharray: 3 2
+}
+.tick text {
+	font-size: 14px
+}
+.x.axis text {
+	font-size: 12px;
+	text-anchor: start !important
+}
+.p-bar-chart .left-axis {
+	font-size: 1rem;
+	text-align: right
+}
+</style>
+    
+<section class="p-strip--suru is-dark">
+    <div class="row u-equal-height fiximg">
+      <div class="col-7">
+        <h1>使用Ubuntu Server横向扩展</h1>
+        <p>Ubuntu Server可让您的公共或私有数据中心在经济和技术上都具有出色的可扩展性。无论是部署OpenStack云、Kubernetes集群还是50000个节点的大型渲染场，Ubuntu Server都能提供性价比最佳的横向扩展能力。</p>
+        <p><a class="p-button--positive" href="/download/">下载Ubuntu服务器版</a></p>
       </div>
-      <div class="col-6 u-vertically-center u-align--center u-hide--small">
-        <img src="https://assets.ubuntu.com/v1/4b47cae5-image-server.svg" width="400" height="121" alt="" />
+      <div class="col-5 u-vertically-center u-align--center u-hide--small ">
+        <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_350,h_121/https://assets.ubuntu.com/v1/55eda3ae-Servers+-white.svg" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_700,h_242/https://assets.ubuntu.com/v1/55eda3ae-Servers+-white.svg 2x" alt="" width="350" height="121" loading="auto">
```

Comment:
These cloudinary image links shouldn't be hard-coded but should be included using the image tag generator. A usage example is:
```
    {{
      image(
          url="https://assets.ubuntu.com/v1/04115a7e-ubuntu_black-orange_hex.svg",
          alt="Ubuntu",
          width="143",
          height="32",
          hi_def=True,
          loading="auto",
      ) | safe
    }}
```